rust/lb-operator/src/main.rs

```diff
@@ -0,0 +1,99 @@
+use std::{os::unix::prelude::FileTypeExt, path::PathBuf};
+
+use clap::Parser;
+use csi_server::{
+    controller::LbOperatorController, identity::LbOperatorIdentity, node::LbOperatorNode,
+};
+use futures::{pin_mut, FutureExt, TryStreamExt};
+use grpc::csi::v1::{
+    controller_server::ControllerServer, identity_server::IdentityServer, node_server::NodeServer,
+};
+use stackable_operator::{kube::CustomResourceExt, logging::TracingTarget};
+use tokio::signal::unix::{signal, SignalKind};
+use tokio_stream::wrappers::UnixListenerStream;
+use tonic::transport::Server;
+use utils::{uds_bind_private, TonicUnixStream};
+
+use crate::crd::{LoadBalancer, LoadBalancerClass};
+
+mod crd;
+mod csi_server;
+mod grpc;
+mod lb_controller;
+mod utils;
+
+#[derive(clap::Parser)]
+#[clap(author, version)]
+struct Opts {
+    #[clap(subcommand)]
+    cmd: stackable_operator::cli::Command<LbOperatorRun>,
+}
+
+#[derive(clap::Parser)]
+struct LbOperatorRun {
+    #[clap(long, env)]
+    csi_endpoint: PathBuf,
+    #[clap(long, env)]
+    node_name: String,
+    #[clap(long, env, default_value_t, arg_enum)]
+    tracing_target: TracingTarget,
+}
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    let opts = Opts::parse();
+    match opts.cmd {
+        stackable_operator::cli::Command::Crd => {
+            println!(
+                "{}{}",
+                serde_yaml::to_string(&LoadBalancerClass::crd()).unwrap(),
+                serde_yaml::to_string(&LoadBalancer::crd()).unwrap()
+            );
+        }
+        stackable_operator::cli::Command::Run(LbOperatorRun {
+            csi_endpoint,
+            node_name,
+            tracing_target,
+        }) => {
+            stackable_operator::logging::initialize_logging(
+                "LB_OPERATOR_LOG",
+                "lb-operator",
+                tracing_target,
+            );
```

Comment:
print the startup string after this as well (stackable_operator::utils::print_startup_string)?

Reply:
4aca95a2f8f182c32800c29b5ee1f84d815a7c34